programs/libraries/pretty_print.sc

```diff
@@ -0,0 +1,129 @@
+// Adds a pretty_print function which is equivalent to the print function, but prints stuff different colors depending on type. Avoids confusion between null and 'null' and between maps, strings, and nbts
+
+// Types & colors
+// null - purple
+// number - blue
+// string - orange
+// list - yellow
+// map - red
+// iterator - Darker yellow
+// function - magenta
+// task - grey
+// block - green
+// entity - Darker red
+// nbt - cyan
+// text - Unchanged, usually formatted already
+
+format_type(v) -> (
```

Comment:
Couldn't you put this in a map? like:
```
type_format_colour_map = {
   'null' -> 'p ',
   'number' -> '#0080ff '
}
```
etc?

Reply:
Ah wait nvm I understand why that wouldnt work for lists and maps, yeah. I mean, you could technically do something with lambda functions etc, but that would be more bulky and less useful.